main.tf

```diff
@@ -115,7 +138,7 @@ EOF
 resource "aws_iam_policy" "sleuth_policy" {
   name        = "aws-iam-sleuth-policy"
   description = "Policy for IAM sleuth lambda checker"
-  policy      = data.aws_iam_policy_document.task_role_policy_doc.json
+  policy      = var.sns_topic_arn == "" ? data.aws_iam_policy_document.task_role_policy_doc[0].json : data.aws_iam_policy_document.task_role_policy_doc_with_sns_topic[0].json
```

Comment:
Ah...I see what you're doing. Do you want to name them *both* explicitly? ie, `task_role_policy_doc_with_slack_webhook` and `task_role_policy_doc_with_sns_topic`? Or put the commonalities in one policy, and then attach the SNS/Slack specific components in separate policies?

Reply:
whoops didn't realize i left that conditional in. should be replaced with the enable one. i don't want to name them both explicitly since one can have both the slack webhook and the sns topic at the same time.